xbmc/guilib/guiinfo/VideoGUIInfo.cpp

```diff
@@ -521,6 +521,24 @@ bool CVideoGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
       case LISTITEM_VIDEO_HDR_TYPE:
         value = tag->m_streamDetails.GetVideoHdrType();
         return true;
+      case LISTITEM_VIDEOVERSION_NAME:
+        value = tag->GetAssetInfo().GetTitle();
+        return true;
+      case LISTITEM_LABEL:
+      {
+        //! @todo get rid of "videos with versions as folder" hack!
+
+        // special casing for "show videos with multiple versions as folders", where the label
+        // should be the video version, not the movie title.
+        CGUIWindow* videoNav{
+            CServiceBroker::GetGUI()->GetWindowManager().GetWindow(WINDOW_VIDEO_NAV)};
```

Comment:
Is it possible to get rid of this somehow?

Reply:
I mean to call into the window. Can't it be done just evaluating if gtassetinfo is set or something?

Reply:
Sorry, no. It is just a hack for Omega. The whole implementation of the feature "show videos  with multiple versions" needs to be revised. That's why I'm putting a respective todo in the code.

Reply:
Alright. It's not that bad since afaik the info interface is gui (skins) anyway. So without gui the info interface is gone too. However even with the coupling  it'd be great if somehow we could avoid calling directly into windows. Hopefully it can be revised later by someone who uses and knows the versions code.